central/complianceoperator/v2/compliancemanager/singleton.go

```diff
@@ -0,0 +1,19 @@
+package compliancemanager
+
+import (
+	"github.com/stackrox/rox/central/sensor/service/connection"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	manager Manager
+	once    sync.Once
+)
+
+// Singleton returns the compliance operator manager
+func Singleton() Manager {
+	once.Do(func() {
+		manager = New(connection.ManagerSingleton())
```

Comment:
nit:  We should probably put this behind the flag `features.ComplianceEnhancements.Enabled()`.

Reply:
I thought about it, and I feel that the checks should be in the caller because`nil` will cause system panic.

Reply:
Fair enough